tests/test_creation/test_relative_features.py

```diff
@@ -403,3 +423,31 @@ def test_get_feature_names_out_raises_error_when_wrong_param(
 
     with pytest.raises(ValueError):
         transformer.get_feature_names_out(input_features=_input_features)
+
+
+def test_transformer_fill_values_when_division_by_zero(df_vartypes):
```

Comment:
I expanded it to test both int and float. I don't see how I can test `None` in this unit test because the transformer will throw an error. `None` should be captured in a different test.